sentinel5dl/__init__.py

```diff
@@ -181,4 +197,22 @@ def download(products, output_dir='.', logger_fn=None):
         if logger_fn:
             logger_fn(f'Downloading {uuid} to {filename}')
         path = f'/odata/v1/Products(\'{uuid}\')/$value'
+
+        # Get md5 sum
+        md5um_path = f'/odata/v1/Products(\'{uuid}\')/Checksum/Value/$value'
```

Comment:
We do not need to make a network request for this if the file does not exist. Moving it down into the if-clause below makes more sense.

Reply:
Main reason here was that we could also change the md5 after the download finished to check if everything was downloaded correctly.

Reply:
Nonetheless, since we do not check the md5 sum after the download, I moved it.

Reply:
Hm… you didn't move it, did you?

Reply:
Yeah sorry my fault. Should now be moved.